x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/oidc/OpenIdConnectRealm.java

```diff
@@ -178,8 +184,21 @@ private void buildUserFromClaims(JWTClaimsSet claims, ActionListener<Authenticat
             return;
         }
 
+        final Map<String, Object> tokenMetadata = new HashMap<>();
+        tokenMetadata.put("id_token_hint", claims.getClaim("id_token_hint"));
```

Comment:
Does this claim exist? I believe not. I think we need to put the JWT id token in here.

Reply:
It;s not a standard claim not. This was handled in https://github.com/elastic/elasticsearch/pull/38475/commits/3e67b2b111abcd66186feb5f8344d48057b7f0da#diff-548117e16c773d2ce359cfabe8ef8a69R222 where the serialized token was added in as a synthetic claim during authentication but I guess this was messed up in a `git merge` effort somehow. I'll reintroduce this and check that nothing else was inadvertently removed